pkg/utils/pretty/changemonitor.go

```diff
@@ -34,6 +34,12 @@ func NewChangeMonitor() *ChangeMonitor {
 	}
 }
 
+// Reconfigure allows reconfiguring the change monitor with a new duration. This resets any previously recorded
+// changes and should only be done at construction.
+func (c *ChangeMonitor) Reconfigure(expiration time.Duration) {
```

Comment:
Why not pass this in the constructor as an option, rather than making it and then mutating it?

Reply:
There were a bunch of other usages already. I figured I would leave 24h as the default and allow changing it post construction.